roles/KubevirtNodeLabeller/tasks/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Copy kubevirt-cpu-node-labeller.yaml to temp directory
```

Comment:
You can replace the temporary file by using lookup('template', "/opt/ansible/roles/KubevirtNodeLabeller/templates/kubevirt-node-labeller-" + version + ".yaml") and k8s.definition. See the last example in https://docs.ansible.com/ansible/latest/modules/k8s_module.html#k8s-module